src/pyfmi/fmi.pyx

```diff
@@ -1501,7 +1501,7 @@ cdef class FMUModelBase(ModelBase):
 
             model.get_version()
         """
-        version = FMIL.fmi1_import_get_version(self._fmu)
+        cdef FMIL.fmi1_string_t version = <FMIL.fmi1_string_t>FMIL.fmi1_import_get_version(self._fmu)
```

Comment:
This and many other places; various returns are FMIL.* strings instead of char* and the explicit typecast suppresses a warning on lost `const`